cookbooks/aws-parallelcluster-slurm/recipes/config_head_node.rb

```diff
@@ -192,34 +192,6 @@
   action :create
 end
 
-if node['cluster']['add_node_hostnames_in_hosts_file'] == "true"
-  directory "#{node['cluster']['slurm']['install_dir']}/etc/pcluster/prolog.d" do
-    user 'root'
-    group 'root'
-    mode '0755'
-  end
-
-  cookbook_file "#{node['cluster']['slurm']['install_dir']}/etc/pcluster/prolog.d/01-pcluster-prolog" do
```

Comment:
you might also want to delete the content of this file

Reply:
Done